### ISSUE TITLE
no option is provided to disable ANSI color escape code

### ISSUE THREAD

**Issue by trombik** [sleede/fab-manager#345]
**Describe the bug**

in rails 5.x, `config.colorize_logging` is `true` by default. users cannot disable ANSI color escape code in the log.

**To Reproduce**

1. run `rails db:seed`
2. See the log

**Expected behavior**

There should be an option to disable color in the log.

**Screenshots**

N/A

**Server (please complete the following information):**
 - OS: FreeBSD 13.0
 - Kernel: FreeBSD 14.0-CURRENT amd64
 - Fab-manager version 5.3.13

**Browser (please complete the following information):**
 - Name: N/A
 - Version: N/A
 
**Additional context**

as an admin, i would like to log to file or syslog. logs in color does not look nice in this context. see an example in color at https://github.com/trombik/ansible-role-fab_manager/runs/6353345075?check_suite_focus=true

**Comment by sylvainbx** [comment]
Thanks for your contribution @trombik. Your PR has been merged and will be released in Fab-manager v5.4.1

**Comment by trombik** [comment]
@sylvainbx thank you. a question. I haven't seen a commit that mentions the PR. some commits seems to go directly into master. how do you manage develop-merge-release workflow?

**Comment by sylvainbx** [comment]
It seems like we forget to put the issue ID in the merge-commit message. Anyway, all changes must go to the `dev` branch or onto a new branch based on `dev` which will be merged on, once the development is over. Only once eveything on `dev` is ready to go to production, we merge the `dev` branch onto `master` and create a new release.

**Comment by trombik** [comment]
@sylvainbx okay.  `PR` > `dev` > `master` looks reasonable. thanks for the clarification.

**Comment by trombik** [comment]
another question: what convention do you follow in commit logs? looks like you prefer prefixes like `(setup)`, `(bug)`, or `(security)`.

i roughly follow the Conventional Commits.

```log
bugfix: fix something

fixes #123. X was replaced with Y because Z.
```

https://www.conventionalcommits.org/en/v1.0.0/

do you have a guidance of possible prefixes or a standard of commit logs?
